src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.html

```diff
@@ -157,3 +170,11 @@
      (click)="resetFilter()"
      i18n>Reset filter.</a>
 </ng-template>
+
+<ng-template #daemonLogsTpl>
+  <cd-alert-panel type="info"
```

Comment:
If you switch to the `Daemon Logs` faster at the first time, you'll see this alert even when the service is enabled. Its because the check for loki/promtail is up or not is still running. So in this situation I think the page needs to wait until it check for the status and return the correct status. We have the same behaviour in services page I guess where we wait until the orch status return something to show/not-show the alert.

Reply:
I couldn't really reproduce this :/, I think we can confirm this behaviour over a call

Reply:
Might've been my environmental issue too. I'll try and reproduce this and if seen I'll definitely share. For now I'll approve then!